.github/workflows/ci.yml

```diff
@@ -35,9 +35,9 @@ jobs:
       - uses: actions/checkout@v4
       - uses: dtolnay/rust-toolchain@master
         with:
-            toolchain: 1.61
-      - run: cargo test --lib --target x86_64-unknown-linux-gnu
-      - run: cargo test --doc --target x86_64-unknown-linux-gnu
+            toolchain: 1.75
+      - run: cargo build --lib --target x86_64-unknown-linux-gnu
+      - run: cargo doc --target x86_64-unknown-linux-gnu
```

Comment:
Do the examples not build with Rust 1.75.0? If so, please rename this step "build with MSRV"

Reply:
Specifically embassy needs 1.79 at the moment; renamed now